src/main/java/Duke.java

```diff
@@ -1,10 +1,96 @@
+import tasks.*;
+import exceptions.*;
```

Comment:
Maybe can name out all the files that you are importing instead of using a wildcard?